solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -242,6 +256,22 @@ public static void main(String[] args) {
         .setDefault(ARG_CLUSTER_ID_DEFAULT)
         .help(ARG_CLUSTER_ID_HELP);
 
+    parser
```

Comment:
Curious if we love this parser or not for CLI args???   THoughts on this versus the commons-cli ?

Reply:
I used what was there already :( I can take a stab at rewriting this cli code on top of commons-cli